mindsdb/api/mysql/mysql_proxy/mysql_proxy.py

```diff
@@ -35,7 +34,7 @@
 from mindsdb.api.mysql.mysql_proxy.classes.server_capabilities import server_capabilities
 from mindsdb.api.mysql.mysql_proxy.classes.sql_statement_parser import SqlStatementParser
 from mindsdb.api.mysql.mysql_proxy.utilities import log
-
+from mindsdb.api.mysql.mysql_proxy.utilities.lightwood_dtype import dtype
```

Comment:
As in `mindsdb_datanode`, we should remove this as a requirement. It's being used in `to_mysql_records`.

Maybe a solution is to enforce on all predictive handlers to always output `numpy` types.